### ISSUE TITLE
0.6 broken (affects purge-db and purge-services)

### ISSUE THREAD

**Issue by mweinreich** [icanhazstring/duck-pony#20]
This happens, when trying to purge databases:
```
[user@machine ~]# /home/application/maintenance/bin/dp issue:purge-db --status="closed" --pattern="/abcd-\d{1,4}/" db1_ db2_ 
In PurgeIssueDatabaseCommand.php line 97:
  MySQL config is incorrect! Username, password and hostname required!
issue:purge-db [-s|--status STATUS] [-i|--invert] [-p|--pattern PATTERN] [--] [<branchname-filter>...]
```

And this is the result of purging services:
```
[user@machine ~]# /home/application/maintenance/bin/dp issue:purge-service /home/application/ --unit=integration_jobworker@
PHP Warning:  preg_match_all(): Compilation failed: invalid range in character class at offset 12 in /home/application/maintenance/vendor/icanhazstring/systemctl-php/src/Utils/OutputFetcher.php on line 22
PHP Stack trace:
PHP   1. {main}() /home/application/maintenance/bin/dp:0
PHP   2. {closure:/home/application/maintenance/bin/dp:33-50}() /home/application/maintenance/bin/dp:50
PHP   3. Symfony\Component\Console\Application->run() /home/application/maintenance/bin/dp:45
PHP   4. Symfony\Component\Console\Application->doRun() /home/application/maintenance/vendor/symfony/console/Application.php:148
PHP   5. Symfony\Component\Console\Application->doRunCommand() /home/application/maintenance/vendor/symfony/console/Application.php:272
PHP   6. duckpony\Console\Command\PurgeIssueServiceCommand->run() /home/application/maintenance/vendor/symfony/console/Application.php:1011
PHP   7. duckpony\Console\Command\PurgeIssueServiceCommand->execute() /home/application/maintenance/vendor/symfony/console/Command/Command.php:255
PHP   8. SystemCtl\SystemCtl->getServices() /home/application/maintenance/src/Console/Command/PurgeIssueServiceCommand.php:78
PHP   9. SystemCtl\SystemCtl->listUnits() /home/application/maintenance/vendor/icanhazstring/systemctl-php/src/SystemCtl.php:157
PHP  10. array_reduce() /home/application/maintenance/vendor/icanhazstring/systemctl-php/src/SystemCtl.php:112
PHP  11. SystemCtl\SystemCtl->SystemCtl\{closure:/home/application/maintenance/vendor/icanhazstring/systemctl-php/src/SystemCtl.php:108-112}() /home/application/maintenance/vendor/icanhazstring/systemctl-php/src/SystemCtl.php:112
PHP  12. SystemCtl\Utils\OutputFetcher::fetchUnitNames() /home/application/maintenance/vendor/icanhazstring/systemctl-php/src/SystemCtl.php:109
PHP  13. preg_match_all() /home/application/maintenance/vendor/icanhazstring/systemctl-php/src/Utils/OutputFetcher.php:22
PHP Warning:  preg_match(): Empty regular expression in /home/application/maintenance/src/Console/Command/PurgeIssueServiceCommand.php on line 80
PHP Stack trace:
PHP   1. {main}() /home/application/maintenance/bin/dp:0
PHP   2. {closure:/home/application/maintenance/bin/dp:33-50}() /home/application/maintenance/bin/dp:50
PHP   3. Symfony\Component\Console\Application->run() /home/application/maintenance/bin/dp:45
PHP   4. Symfony\Component\Console\Application->doRun() /home/application/maintenance/vendor/symfony/console/Application.php:148
PHP   5. Symfony\Component\Console\Application->doRunCommand() /home/application/maintenance/vendor/symfony/console/Application.php:272
PHP   6. duckpony\Console\Command\PurgeIssueServiceCommand->run() /home/application/maintenance/vendor/symfony/console/Application.php:1011
PHP   7. duckpony\Console\Command\PurgeIssueServiceCommand->execute() /home/application/maintenance/vendor/symfony/console/Command/Command.php:255
PHP   8. Symfony\Component\Finder\Iterator\PathFilterIterator->rewind() /home/application/maintenance/src/Console/Command/PurgeIssueServiceCommand.php:84
PHP   9. Symfony\Component\Finder\Iterator\CustomFilterIterator->rewind() /home/application/maintenance/src/Console/Command/PurgeIssueServiceCommand.php:84
PHP  10. Symfony\Component\Finder\Iterator\CustomFilterIterator->accept() /home/application/maintenance/src/Console/Command/PurgeIssueServiceCommand.php:84
PHP  11. duckpony\Console\Command\PurgeIssueServiceCommand->duckpony\Console\Command\{closure:/home/application/maintenance/src/Console/Command/PurgeIssueServiceCommand.php:79-81}() /home/application/maintenance/vendor/symfony/finder/Iterator/CustomFilterIterator.php:54
PHP  12. preg_match() /home/application/maintenance/src/Console/Command/PurgeIssueServiceCommand.php:80
```

**Comment by icanhazstring** [comment]
Hi @mweinreich thanks for reporting. Will look into this as soon as I got the time :)

**Comment by icanhazstring** [comment]
Solved with #21 and tagged 0.6.1

**Comment by smuggli** [comment]
Another warning has been fixed in #22 and tagged 0.6.2
